server/test/repository/ApplicationStatusesRepositoryTest.java

```diff
@@ -79,20 +80,22 @@ public void canQueryForAllApplicationStatuses() {
 
     assertThat(statusDefinitionsModelResults).isNotEmpty();
     // one status is added as part of the program creation and one status as obsolete status
-    assertThat(statusDefinitionsModelResults.size()).isEqualTo(3);
+    assertThat(statusDefinitionsModelResults.size()).isEqualTo(4);
     assertThat(statusDefinitionsModelResults.get(0).getStatusDefinitions().getStatuses().size())
+        .isEqualTo(0);
+    assertThat(statusDefinitionsModelResults.get(2).getStatusDefinitions().getStatuses().size())
```

Comment:
**question:** What does `get(1).getStatusDefinitions()
                .getStatuses()
                .get(0)
                .statusText())` return?

Reply:
there won't be any statuses as its empty on program creation. I added a line to test it as well provided detail explanation in the unit tests- 
`    assertThat(statusDefinitionsModelResults.get(0).getStatusDefinitions().getStatuses().size())
        .isEqualTo(0);`